roles/ceph-common/tasks/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+- include: check_system_and_vars.yml
+
+- name: install dependencies
+  apt: pkg={{ item }}
+       state=present
+       update_cache=yes
+       cache_valid_time=3600
+  with_items: ceph_common.debian_pkgs
+
+- name: install the ceph repository stable key
```

Comment:
the key and repo should be added in the apt-repos role and added as a dependency to this role. see sensu and rabbit etc on how that is done.